actionview/test/template/sanitize_helper_test.rb

```diff
@@ -37,6 +37,12 @@ def test_strip_tags_will_not_encode_special_characters
     assert_equal "test\r\n\r\ntest", strip_tags("test\r\n\r\ntest")
   end
 
+  def test_strip_tags_is_marked_safe
+    frag = "<div>&lt;<span>script</span>&gt;xss();&lt;<span>/script</span>&gt;</div>"
+    assert_equal("&lt;script&gt;xss();&lt;/script&gt;", strip_tags(frag)) # this string is safe for use as pcdata in html
+    assert_predicate(strip_tags(frag), :html_safe?)
+  end
```

Comment:
Perhaps for further confidence, we could add a few more cases to `test_strip_tags` above.  e.g.:

```ruby
assert_equal "firstname", strip_tags("firstname<script")
assert_equal "middlename&gt;xss()", strip_tags("middlename>xss()")
assert_equal "lastname", strip_tags("lastname</script>")
```

Reply:
This behavior is well-tested in rails-html-sanitizer and the `assert_equal` here is more for the reader of the test than anything else. I don't think it's necessary to add more coverage here, but will do so if you feel strongly about it.